mautic/mautic
# WIP: Fix the global page search issue

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


An unexpected error happening during a global search pages to an error: Call to a member function on array

![image](https://user-images.githubusercontent.com/96085911/188419046-ff49ca3c-31c8-43cb-9a00-da7d2153ad9c.png)
#### Steps to reproduce:
1. Create a new landing page
2. Try to use global search to find it
3. There is an error

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. The same steps as when reproducing the issue
3. There is no error
